back-end/db/migrations/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"log"
+
+	"../schema"
+	"../seeds"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	db, err := gorm.Open("postgres", "host=127.0.0.1 port=5432 user=sap dbname=postgres password='password' sslmode=disable")
```

Comment:
Does Go have the equivalent of `dotenv`?

Reply:
It does, I'll get that changed over and then update the request.